spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -201,6 +201,7 @@ public long splitSize() {
     return confParser
         .longConf()
         .option(SparkReadOptions.SPLIT_SIZE)
+        .sessionConf(SparkSQLProperties.SPLIT_SIZE)
```

Comment:
Could we have a test for this?
So future changes will not remove/revert the change accidentally?